### PR TITLE
refactor claws-mail and sylpheed as whitelist profiles

### PR DESCRIPTION
**Pull request by glitsj16**
Both profiles should benefit from being whitelist profiles IMHO. A review would be appreciated.

### CODE REVIEW

File: etc/email-common.profile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# added by caller profile
+#include globals.local
+
+noblacklist ${HOME}/.gnupg
+noblacklist ${HOME}/.signature
+# when storing mail outside the default ${HOME}/Mail path, 'noblacklist' the custom path in your email-common.local
+# and 'blacklist' it in your disable-common.local too so it is  kept hidden from other applications
+noblacklist ${HOME}/Mail
+
+noblacklist ${DOCUMENTS}
```
Comment (rusty-snake) [inline etc/email-common.profile]: Add `include disable-xdg.inc` or remove it.

**Comment by glitsj16** [comment]
@rusty-snake What was eaten by FitHub? Would you mind repeating it please?

File: etc/email-common.profile (hunk 1 above)
Comment (glitsj16) [reply etc/email-common.profile]: Fixed, thanks.

**Comment by rusty-snake** [comment]
In the first (approve) review I added this commend, but it's lost in ~space~ internet.
> Add `include disable-xdg.inc` or remove it.

**Pull request merged**